src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,33 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    static final String SEARCH_LETTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        ArrayList<String> list = new ArrayList<String>();
+        ArrayList<String> resultList = new ArrayList<String>();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileName));
```

Comment:
do we need to close `reader` in the end?

Reply:
done